src/assets/partials/_contact.scss

```diff
@@ -0,0 +1,88 @@
+.contact-title {
+	color: #FFFFFF; //TODO variables
```

Comment:
Please use the vars. Refer to [_vars.scss](https://github.com/peterxqe/teamfairtrade/blob/develop/src/assets/modules/_vars.scss) for correct var names.